lisa/sut_orchestrator/azure/common.py

```diff
@@ -251,6 +251,7 @@ class AzureNodeSchema:
     # It decides the real computer name. It cannot be too long.
     short_name: str = ""
     vm_size: str = ""
+    resource_sku: Optional[Dict[str, Any]] = None
```

Comment:
The resource sku shouldn't saved in the node scheme. You can use the logic to read it from the local cache in the azure platform.

Reply:
I didn't move `_load_location_info_from_file` to common.py as we discussed because that brings a circle import error for `AzureLocation` is defined in platform_.py.

I copied `_load_location_info_from_file` to notifiers.py.

Reply:
The AzureLocation should be able to move to common.py. Copy code is not recommended.

Reply:
Done